draft-ietf-quic-transport.md

```diff
@@ -3557,12 +3557,18 @@ which could prevent the connection from ever becoming idle.  Non-ack-eliciting
 packets are eventually acknowledged when the endpoint sends an ACK frame in
 response to other events.
 
-In order to assist loss detection at the sender, an endpoint SHOULD send an ACK
-frame immediately on receiving an ack-eliciting packet that is out of order. The
-endpoint SHOULD NOT continue sending ACK frames immediately unless more
-ack-eliciting packets are received out of order.  If every subsequent
-ack-eliciting packet arrives out of order, then an ACK frame SHOULD be sent
-immediately for every received ack-eliciting packet.
+In order to assist loss detection at the sender, an endpoint SHOULD generate
+and send an ACK frame without delay when it receives an ack-eliciting packet
+either:
+
+* When the received packet has a packet number less than another ack-eliciting
+  packet that has been received.
+* When the packet has a packet number larger than any other ack-eliciting packet
+  that has been received and there are missing packets between that other packet
+  and this packet.
+
+The endpoint SHOULD NOT continue sending ACK frames immediately unless one of
+the above criteria are satisfied.
```

Comment:
This is an effort to keep the existing text, but rework it to refer to the criteria that's been reworked to no longer mention out of order.

Specifically, the existing text is "The endpoint SHOULD NOT continue sending ACK frames immediately unless more ack-eliciting packets are received out of order."

The goal of this text is: you should send one immediate ACK if the above happens, but don't keep sending immediate ACKs for every subsequent ack-eliciting packet, because that would be excessive.

Suggestions welcome.